services/net/content/ContentManager.cs

```diff
@@ -221,7 +225,48 @@ private void ListenerStopHandler(object sender, EventArgs e)
     /// <exception cref="InvalidOperationException"></exception>
     private async Task HandleMessageAsync(ConsumeResult<string, SourceContent> result)
     {
-        this.Logger.LogDebug("Importing Content from Topic: {topic}, Uid: {key}", result.Topic, result.Message.Key);
+        try
+        {
+            // The service has stopped, so to should consuming messages.
+            if (this.State.Status != ServiceStatus.Running)
+            {
+                this.Listener.Stop();
+                this.State.Stop();
+            }
+            else
+            {
+                await ProcessSourceContentAsync(result);
+
+                // Inform Kafka this message is completed.
+                this.Listener.Commit(result);
```

Comment:
If we're not pausing do we need to commit here?

Reply:
I was thinking that as well, but I left it in place as that's the same patter as the notification service uses.

My guess is that Kafka will ignore the commit because we have told it to autocommit.  If we could determine the config of the Listener at runtime we could wrap that `.Commit(` in an `if` statement.  Because that flag is part of the Kafka config as opposed to service Options, there didn't seem to be an easy way to check...